js/status-page.js

```diff
@@ -18,6 +18,26 @@
     }
   };
 
+  const onFormElementClick = (evt) => {
+    switch (evt.target) {
+      case window.form.adNode.rooms:
+        evt.target.addEventListener(`input`, window.form.validateRoomsInput);
+        break;
+      case window.form.adNode.title:
+        evt.target.addEventListener(`input`, window.form.validateTitleInput);
+        break;
+      case window.form.adNode.type:
+        evt.target.addEventListener(`input`, window.form.validateTypeInput);
+        break;
+      case window.form.adNode.timein:
+        evt.target.addEventListener(`input`, window.form.validateTimeInput);
+        break;
+      case window.form.adNode.timeout:
+        evt.target.addEventListener(`input`, window.form.validateTimeInput);
+        break;
```

Comment:
- Не работает потому, что после change ты должен вызывать колбеки, а не обработчики навешивать. В текущем решении у тебя будут навешиваться обработчики после каждого срабатывания
- Так могут копиться одинаковые обработчики, а это утечка памяти
-  Кейсы с одинаковым исходом можно группировать

```suggestion
      case window.form.adNode.timein:
      case window.form.adNode.timeout:
        window.form.validateTimeInput();
        break;
```